packages/vuetify/src/styles/settings/_dark.scss

```diff
@@ -65,9 +65,8 @@ $material-dark: (
     'color': map-get($shades, 'white')
   ),
   'text-fields': (
-    'box': rgba(map-get($shades, 'black'), 0.1),
-    'box-hover': rgba(map-get($shades, 'black'), 0.2),
-    'box-focus': rgba(map-get($shades, 'black'), 0.3)
```

Comment:
why is 'filled-focus' missing on this new update?

Reply:
not required